src/ui/pages/NotificationDetails/components/CredentialRequest/CredentialRequest.test.tsx

```diff
@@ -161,7 +161,7 @@ describe("Credential request", () => {
   });
 });
 
-describe("Credential request: Multisig", () => {
+describe.skip("Credential request: Multisig", () => {
```

Comment:
@sdisalvo-crd @Sotatek-DukeVu Please note I skipped this test suite, since the UI will change so much here in the next UI PR.